src/main/java/net/irisshaders/lilybot/commands/support/UpdateDrivers.java

```diff
@@ -11,41 +10,96 @@
-                .setTitle("Update your Drivers.")
-                .setDescription(
-                        """
-                                This can be done in multiple ways, depending on your graphics card:
-                                
-                                AMD: You can use AMD's Radeon Software to easily update your drivers, If you don't have it, download it here: https://www.amd.com/en/support
-                                
-                                NVIDIA: You can use GeForce Experience to easily update drivers, If you don't have it, download it here: https://www.nvidia.com/Download/index.aspx?lang=us
-                                
-                                Intel: You can use Intel DSA to easily update your drivers, If you don't have it, download it here: https://www.intel.com/content/www/us/en/support/detect.html"""
-                )
+        public NvidiaDrivers() {
+            this.name = "nvidia";
+            this.help = "Informs the user that they should try to update their Nvidia drivers";
+            this.defaultEnabled = true;
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(SlashCommandEvent event) {
+            User user = event.getUser();
+
+            event.replyEmbeds(driverUtils(user, "Update your Nvidia Drivers", DriverDescriptions.NvidiaDrivers))
+                    .mentionRepliedUser(false).setEphemeral(false).queue();
+        }
+    }
+
+    public static class IntelDrivers extends SlashCommand {
+
+        public IntelDrivers() {
+            this.name = "intel";
+            this.help = "Informs the user that they should try to update their Intel drivers";
+            this.defaultEnabled = true;
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(SlashCommandEvent event) {
+            User user = event.getUser();
+
+            event.replyEmbeds(driverUtils(user, "Update your Intel Drivers", DriverDescriptions.IntelDrivers))
+                    .mentionRepliedUser(false).setEphemeral(false).queue();
+        }
+    }
+
+    private static MessageEmbed driverUtils(User user, String title, String description) {
+
+        return new EmbedBuilder()
+                .setTitle(title)
+                .setDescription(description)
                 .setColor(Color.RED)
                 .setFooter("Requested by " + user.getAsTag(), user.getEffectiveAvatarUrl())
                 .setTimestamp(Instant.now())
                 .build();
-        event.replyEmbeds(embed).mentionRepliedUser(false).setEphemeral(false).queue();
+    }
+
+    private interface DriverDescriptions {
+
+        String AMDDrivers = "You can use AMD's Radeon Software to easily update your drivers, If you don't have it, " +
+                "download it here: https://www.amd.com/en/support";
+        String NvidiaDrivers = "You can use GeForce Experience to easily update drivers, If you don't have it, " +
```

Comment:
Should be Nvidia GeForce Experience to be consitent

Reply:
done